draft-pda-protocol.md

```diff
@@ -937,13 +918,14 @@ criteria:
    is the prime modulus.
 1. **Fast polynomial operations.** In order to make Prio practical, it's
    important that implementations employ FFT to speed up polynomial operations.
-   In particular, the prime modulus p should be chosen so that (p-1) = 2^b * s
-   for large b and odd s. Then g^s is a principle, 2^b-th root of unity (i.e.,
-   g^(s\*2^b) = 1), where g is the generator of the multiplicative subgroup.
-   This fact allows us to quickly evaluate and interpolate polynomials at 2^a-th
-   roots of unity for any 1 <= a <= b. Note that b imposes an upper bound on the
-   size of proofs, so it should be large enough to accommodate all foreseeable
-   use cases. Something like b >= 20 is probably good enough.
+   In particular, the prime modulus p should be chosen so that `(p-1) = 2^b * s`
+   for large `b` and odd `s`. Then `g^s` is a principle, `2^b`-th root of unity
+   (i.e., `g^(s\*2^b) = 1`), where `g` is the generator of the multiplicative
+   subgroup.
+   This fact allows us to quickly evaluate and interpolate polynomials at
+   `2^a`-th roots of unity for any `1 <= a <= b`. Note that `b` imposes an upper
+   bound on the size of proofs, so it should be large enough to accommodate all
+   foreseeable use cases. Something like `b >= 20` is probably good enough.
```

Comment:
What changed here? Looks like we're just inserting a line break?

Reply:
I'm putting `` around the math terms to make them fixed-width. I find it more readable if those stand out from the regular text, but I'm happy to revert.

Reply:
Ack, no need!